src/app/src/components/UserProfile.jsx

```diff
@@ -250,7 +255,7 @@ function mapStateToProps({
 }) {
     return {
         user,
-        fetching: fetching || sessionFetching,
+        fetching: fetching || sessionFetching || authFetching,
```

Comment:
The changes here are to fix a bug which was surfaced by updating how translation works: namely, there's a moment when `profile` doesn't exist and the form fields shouldn't try to render at all, but they were and it threw an error.

I can move this out into another commit if it would make more sense to have it separated.